app/background.js

```diff
@@ -36,6 +39,7 @@ class BackgroundWorker {
       (details) => {
         this.data[tabId].count += 1;
         chrome.browserAction.setBadgeText({text: '' + this.data[tabId].count, tabId: tabId});
```

Comment:
This is not modified in this PR, but interpolation would be better IMO. Babel has support afaik